server/src/internalClusterTest/java/org/elasticsearch/indices/memory/breaker/HierarchyCircuitBreakerTelemetryIT.java

```diff
@@ -142,37 +97,29 @@ public void testCircuitBreakerTripCountMetric() {
         fail("Expected exception not thrown");
     }
 
-    private List<Measurement> getMeasurements(String dataNodeName) {
-        final TestTelemetryPlugin dataNodeTelemetryPlugin = internalCluster().getInstance(PluginsService.class, dataNodeName)
-            .filterPlugins(TestCircuitBreakerTelemetryPlugin.class)
+    @After
+    public void resetClusterSetting() {
+        final var circuitBreakerSettings = Settings.builder()
+            .putNull(FIELDDATA_CIRCUIT_BREAKER_LIMIT_SETTING.getKey())
+            .putNull(FIELDDATA_CIRCUIT_BREAKER_OVERHEAD_SETTING.getKey())
+            .putNull(REQUEST_CIRCUIT_BREAKER_LIMIT_SETTING.getKey())
+            .putNull(REQUEST_CIRCUIT_BREAKER_OVERHEAD_SETTING.getKey())
+            .putNull(IN_FLIGHT_REQUESTS_CIRCUIT_BREAKER_LIMIT_SETTING.getKey())
+            .putNull(IN_FLIGHT_REQUESTS_CIRCUIT_BREAKER_OVERHEAD_SETTING.getKey())
+            .putNull(TOTAL_CIRCUIT_BREAKER_LIMIT_SETTING.getKey())
+            .putNull(HierarchyCircuitBreakerService.USE_REAL_MEMORY_USAGE_SETTING.getKey());
+        updateClusterSettings(circuitBreakerSettings);
+    }
+
+    private List<Measurement> getMeasurements(String nodeName) {
+        final TestTelemetryPlugin telemetryPlugin = internalCluster().getInstance(PluginsService.class, nodeName)
+            .filterPlugins(TestTelemetryPlugin.class)
             .toList()
             .get(0);
         return Measurement.combine(
-            Stream.of(dataNodeTelemetryPlugin.getLongCounterMeasurement(CircuitBreakerMetrics.ES_BREAKER_TRIP_COUNT_TOTAL).stream())
+            Stream.of(telemetryPlugin.getLongCounterMeasurement(CircuitBreakerMetrics.ES_BREAKER_TRIP_COUNT_TOTAL).stream())
                 .flatMap(Function.identity())
                 .toList()
         );
     }
-
-    // Make sure circuit breaker telemetry on trip count reports the same values as circuit breaker stats
-    private void assertCircuitBreakerTripCount(
```

Comment:
This method was unused, likely a leftover from before adopting `TestTelemetryPlugin`